smoke-tests/seattle/build-config.json

```diff
@@ -32,6 +32,21 @@
       "feedId" : "kcm",
       "source" : "https://metro.kingcounty.gov/GTFS/google_transit.zip"
     },
+    {
+      "type" : "gtfs",
+      "feedId" : "pierce",
+      "source" : "https://gtfs.sound.obaweb.org/prod/3_gtfs.zip"
+    },
+    {
+      "type" : "gtfs",
+      "feedId" : "soundtransit",
+      "source" : "https://gtfs.sound.obaweb.org/prod/40_gtfs.zip"
+    },
+    {
+      "type" : "gtfs",
+      "feedId" : "commtrans",
+      "source" : "https://picsapi.commtrans.org/download/liveGTFS.zip"
```

Comment:
I've always wondered about this host name. What is "picsapi"? A pictures API?

Reply:
I asked this once I think it's something crazy like passenger information collation system